Library/Tests/ViewModels/LiveStreamCountdownViewModelTests.swift

```diff
@@ -145,7 +145,7 @@ internal final class LiveStreamCountdownViewModelTests: TestCase {
 
     XCTAssertTrue(self.pushLiveStreamViewControllerLiveStream.lastValue?.isLiveNow ?? false)
     XCTAssertTrue(self.pushLiveStreamViewControllerProject.lastValue?.liveStreams.first?.isLiveNow ?? false)
-    XCTAssertTrue(self.pushLiveStreamViewControllerEvent.lastValue?.stream.liveNow ?? false)
+    XCTAssertTrue(self.pushLiveStreamViewControllerEvent.lastValue??.stream.liveNow ?? false)
```

Comment:
So was just chatting to @mbrandonw about it. This an optional optional 😄 . Basically because `lastValue` is optional and the third component of the tuple is `LiveStreamEvent?` (also optional) it's optionally unwrapping both in succession. However, I think it's good that you pointed this out, I'm going to instead not use the tuple type in this particular case because the `pushLiveStreamViewControllerEvent` signal will never emit if its LiveStreamEvent is nil.